src/test/regress/spec/isolation_logical_replication_single_shard_commands.spec

```diff
@@ -2,9 +2,6 @@
 // so setting the corresponding shard here is useful
 setup
 {
-  SELECT citus_internal.replace_isolation_tester_func();
```

Comment:
do we still need functions replace_isolation_tester_func / refresh_isolation_tester_prepared_statement  anywhere?

Can we drop those?

Reply:
I think we can not drop them, at least not yet.

We need them in 4 places:
- `isolation_setup.spec` that is the first test run in all isolation schedules. It replaces the isolation tester function before any tests are run.
- The tests that fail when we use citus isolation tester functions, instead of postgres implementation. Namely: 
  - `isolation_citus_dist_activity.spec`
  - `isolation_master_update_node.spec`
  - `isolation_update_node.spec`

The errors on those tests are written in the description for #5964 body.

Reply:
Ah, sorry my bad. Thanks for the clear answer